pkg/util/tls/tls.go

```diff
@@ -13,9 +13,11 @@ import (
 
 // ClientConfig is the config for client TLS.
 type ClientConfig struct {
+	Enable             bool   `yaml:"tls_enable"`
```

Comment:
I would name it `tls_enabled` (more consistent with the rest of the config, even if in ancient times "enable" has been used too). Same comment the the CLI flag (`tls-enabled`).

As a consequence, I would rename the struct field to `Enabled`.